.github/workflows/add_to_triage_project.yml

```diff
@@ -0,0 +1,20 @@
+name: Add issue/PR to project
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-to-triage-project:
+    name: Add to triage project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
```

Comment:
Does it automatically add the issue to the 'new issue' column?

Reply:
Yes, though that is configured in the Project workflow

Reply:
Nice, i see that there now that i look.